README.md

```diff
@@ -36,71 +36,71 @@ $ maximodev-cli dist
-`create script-field-validator`, like the `java-field-valiator`, creates a field validator and registers it to an object and field.  The difference being that this field valiator uses the `automation scripting` framework and does not require Java.
+The `create script-field-validator` command, like the `java-field-valiator` command, creates a field validator and registers it to an object and field. The difference is that this field valiator uses the `automation scripting` framework and does not require Java.
 
 ### maximodev-cli create sample-classic-app
-`create sample-classic-app` prompts you for some information and creates a new sample application in your add-on location.  This sample app is mainly used to help you scaffold a new application, or for you to see how to build a complete working sample application within your add-on.  By using `create sample-classic-app`, dbc scripts and Java files are added, and creates a fully working application.  You can then `build` and `deploy` your add-on to see the application working within your Maximo environment.
+The `create sample-classic-app` command prompts you for some information and creates a new sample application in your add-on location. This sample app is mainly used to help you scaffold a new application or for you to see how to build a complete working sample application in your add-on. By using the `create sample-classic-app`command, the dbc scripts and Java files are added, and a fully working application is created. You can then build and deploy your add-on to see the application working in your Maximo environment.
 
 ### maximodev-cli update product-xml
-`update product-xml` is a command that updates the last script number in your product XML by inspecting and finding the last script in your product scripts area.  This command assumes that your product XML is a `template` file, which was created by  `maximodev-cli`.
+The `update product-xml`command updates the last script number in your product XML file by inspecting and finding the last script in your product scripts area. This command assumes that your product XML file is a `template` file that was created by a `maximodev-cli` command.
 
 ### maximodev-cli set
-`maximodev-cli set` is a command that can be used to update your `addon.properties`.  If you need to set or change your Maximo home page, you can use the following command `maximodev-cli set maximo-home`.  The `set` command prompts a question, and then proceed to update the `addon.properties`.  Currently, only setting or updating the `maximo_home` is supported, but in the future more options may be added.
+The `maximodev-cli set` command can be used to update your `addon.properties` file. If you need to set or change your Maximo home page, you can use the `maximodev-cli set maximo-home` command. The `set` command prompts a question and then updates the `addon.properties` file.  Currently, only setting or updating the `maximo_home` is supported.
```

Comment:
line 76
for legal reasons, avoid ever mentioning future support of anything, even vaguely.